Gemfile.lock

```diff
@@ -47,27 +47,27 @@ GEM
     erubis (2.7.0)
     execjs (2.2.2)
     hike (1.2.3)
-    i18n (0.6.11)
+    i18n (0.7.0)
```

Comment:
Why do we need to update i18n? This is for localization and we're locked on 0.6.11 for `reasons`